### ISSUE TITLE
add security considerations section

### ISSUE THREAD

**Issue by samuelweiler** [mathml-refresh/mathml#236]
Separate from privacy considerations, per guidance in https://w3ctag.github.io/security-questionnaire/

**Comment by davidcarlisle** [comment]
yes we should meet current w3c guidelines for the spec structure.
